jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/test/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImplTest.java

```diff
@@ -55,4 +91,60 @@ private void assertMethodResult(final Method method, final Class<?> returnType,
         }
     }
 
+    @Test
+    public void testGetProcesses(){
+        final ProcessDefinition def = ProcessDefinition.builder().id("processId").build();
+
+        when(queryServicesClient.findProcesses(0, 0, "", true)).thenReturn(singletonList(def));
+
+        final List<ProcessSummary> summaries = service.getProcesses("serverTemplateId", 0, 0, "", true);
+
+        assertNotNull(summaries);
+        assertEquals(1, summaries.size());
+        assertProcessSummary(def, summaries.get(0));
+    }
+
+    @Test
+    public void testGetProcessesByFilter(){
+        final ProcessDefinition def = ProcessDefinition.builder().id("processId").build();
+
+        when(queryServicesClient.findProcesses("filter", 0, 0, "", true)).thenReturn(singletonList(def));
```

Comment:
Since the arguments in findProcesses method, in this test and the previous one, don't play any role in test, maybe switching from hard-coded arguments to arbitrary values makes sense i.e. findProcesses(anyString(), anyInt(), anyInt(), anyString(), anyBoolean())